src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 px 2 px 4 px 0 px rgba(0, 0, 0, 0.05);
```

Comment:
<img width="273" alt="Screenshot 2023-07-04 at 21 16 48" src="https://github.com/mate-academy/layout_moyo-header/assets/117587463/e7c78e88-76e0-4ebf-9a16-327a9bb1aeb8">

```suggestion
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
```